xpdacq/Ni24.D

```diff
@@ -0,0 +1,27 @@
+# Calibrant: Nickel (Ni)
```

Comment:
Yes, that was the implementation detail I didn't fully cover at that time.

Btw, this is a customized d-spacing file made by @sbanerjee23 which capture additional rings. Just want to nominate him :p